routes/chorals.js

```diff
@@ -32,13 +33,23 @@ router.get('/', function(req, res, next) {
 router.post('/', function(req, res, next) {
   var googleUser = req.user;
   var userModel = res.locals.userModel;
+  var cookies = req.get("Cookie");
+  var children = [];
+  var child; 
+
+  for(var i = 0; cookieHelper.readCookie('child' + i, cookies) != null; i++){
```

Comment:
For temp storage of children to be added to parent choral (So that they can be displayed as you add them). Wasn't sure how to do it a different way so I used cookies.